tests/integration/client/feedback/training/test_trainer.py

```diff
@@ -43,8 +45,8 @@
 @pytest.mark.parametrize(
     "framework",
     [
-        Framework("spacy"),
-        Framework("spacy-transformers"),
+        # Framework("spacy"),
+        # Framework("spacy-transformers"),
```

Comment:
Shouldn't we just remove those if not ran?

```suggestion
```

Reply:
Oh, oops. I think these should run as they work on the CI - they just cause my pytest locally to apruptly terminate without any message, so I comment them out.

Reply:
c612fc4516790b17604b5a1ff3d1b77c9ebbaf09